run.sh

```diff
@@ -40,7 +40,8 @@ report_code_coverage()
     # https://docs.codecov.com/docs/codecov-uploader
     printf '*****************************************************************\n'
     printf 'Run the unit tests for all subdirectories\n'
-    pip3 install coverage
+    # docker will require that pip3 forcefully install
+    # pip3 install --break-system-packages coverage
```

Comment:
can be removed?

Reply:
I moved a line to requirements.txt, which I feel will document it well enough for local use.